napari/layers/_tests/test_layer_attributes.py

```diff
@@ -134,6 +136,19 @@ def test_get_value_3d_view_of_2d_image(ImageClass):
     _check_subpixel_values(layer, val_dict)
 
 
+@pytest.mark.parametrize('Layer, data, _', layer_test_data)
+def test_layer_unique_id(Layer, data, _):
+    layer = Layer(data)
+    assert layer.unique_id is not None
+    assert type(layer.unique_id) is uuid.UUID
```

Comment:
Tests should ideally only test that the public API promises are maintained. In this case, this is testing an implementation detail — this test should only test that the type is hashable.